Content.Server/Mech/Systems/MechSystem.cs

```diff
@@ -23,6 +23,7 @@
 using Robust.Shared.Containers;
 using Robust.Shared.Player;
 using Content.Shared.Whitelist;
+using Content.Shared.SS220.MechRobot; ////SS220-AddMechToClothing
```

Comment:
Переборщил с слэшами
```suggestion
using Content.Shared.SS220.MechRobot; //SS220-AddMechToClothing
```